dimagi/pages/templates/partials/footer.html

```diff
@@ -4,68 +4,119 @@
+              </span>
+              {% blocktrans %}
+                Contact us
+              {% endblocktrans %}
             </a>
           </li>
         </ul>
-        <p class="b-corp-logo align-center align-right-at-small">
-          <a href="https://bcorporation.net/"
-             target="_blank">
-            {% include "svg/b_corp_logo.html" %}
-          </a>
-        </p>
       </div>{# .column #}
-
-      <div class="column">
-        <p class="copyright align-center align-left-at-small">
-          &copy; {% now "Y" %} Dimagi, Inc.
+      <div class="column align-center align-left-at-medium">
+        <p>
+          Over 32,000 leaders from WHO, Save the children, the World  Bank. USAID,
+          the Bill and Melinida Gates Foundation, and many Other impact-driven organzations subscribe
+          to our newsletter for case studies, best practices, and other updates. Join then today!
+        </p>
+        <p>
+          <div id="email-subscribtion-form-content"></div>
         </p>
       </div>{# .column #}
-    </div>{# .grid #}
-
+      <div class="column align-center align-right-at-extra-large">
+        <div class="grid-2-at-extra-large">
+          <div class="column">
+            <a href="https://bcorporation.net/"
+               target="_blank"
+               class="b-corp-logo">
+              {% include "svg/b_corp_logo.html" %}
+            </a>
+          </div> {# .column #}
+          <div class="column">
+            <ul class="reset links-social">
+              <li>
+                <a href="//facebook.com/dimagi.inc"
+                   class="{% click_footer 'facebook' %}"
+                   target="_blank">
+                  {% include 'svg/footer/facebook.html' %}
+                </a>
+              </li>
+              <li>
+                <a href="//twitter.com/dimagi"
+                   class="{% click_footer 'twitter' %}"
+                   target="_blank">
+                  {% include 'svg/footer/twitter.html' %}
+                </a>
+              </li>
+              <li>
+                <a href="//linkedin.com/company/dimagi"
+                   class="{% click_footer 'linkedin' %}"
+                   target="_blank">
+                  {% include 'svg/footer/linkedin.html' %}
+                </a>
+              </li>
+              <li>
+                <a href="//youtube.com/channel/UCt8JcRhWywkVJRR_YWv4OhA"
+                   class="{% click_footer 'youtube' %}"
+                   target="_blank">
+                  {% include 'svg/footer/youtube.html' %}
+                </a>
+              </li>
+            </ul>
+            <p class="copyright align-center align-left-at-extra-large">
+              &copy; {% now "Y" %} Dimagi, Inc.
+            </p>
+          </div> {# .column #}
+        </div> {# .grid-2 #}
+      </div>{# .column #}
+    </div>{# .grid-3 .destination-links #}
   </div>{# .content #}
 </footer>
+
+{% block js %}
```

Comment:
`js` blocks are not allowed in partials. since this is something that appears on every page, please do something similar to how the `demoForm` is implemented by including it in the `app.js` library.